docs/src/extend/languages.md

```diff
@@ -124,16 +124,18 @@ In order to use a language from a plugin in a configuration file, import the plu
 
 ```js
 // eslint.config.js
+import { defineConfig } from "eslint/config";
 import example from "eslint-plugin-example";
 
-export default [
+export default defineConfig([
     {
+        files: ["**/*.js"],
```

Comment:
```suggestion
        files: ["**/*.txt"],
```

It would probably make more more sense to create a custom language for something that isn't js.